techniques/system/common/1.0/cf-serverd.st

```diff
@@ -46,10 +46,12 @@ bundle server access_rules
     &MANAGED_NODES_NAME, MANAGED_NODES_ID, MANAGED_NODES_KEY : {host, uuid, key |
       "/var/rudder/share/&uuid&/"
         maproot => {  @{def.acl}  },
+        if => isdir("/var/rudder/share/&uuid&/"),
         admit_keys => { "&key&" };
 
       "/var/rudder/shared-files/&uuid&/"
         maproot => {  @{def.acl}  },
+        if => isdir("/var/rudder/shared-files/&uuid&/"),
```

Comment:
This can't work as this folder is created by the relay which does not reload cf-serverd